core/domain/exp_services.py

```diff
@@ -1500,7 +1500,7 @@ def accept_suggestion(editor_id, thread_id, exploration_id, commit_message):
              suggestion_author_userid, exploration_id, change_list, 
              full_commit_message)
         feedback_services.create_message(
-            exploration_id, thread_id, suggestion_author_userid, None, None,
+            exploration_id, thread_id, editor_id, None, None,
```

Comment:
Ooh, good catch!

Also, it's an unrelated nit, but could you change suggestion_author_userid to suggestion_author_id elsewhere in this method, just to be consistent (since we don't use 'userid' anywhere)? The alternative is suggestion_author_user_id but that's a bit long.